src/app/container/email.service.ts

```diff
@@ -60,8 +29,11 @@ export class EmailService {
    * @param path The path of the tool whose access is being requested
    * @param toolRegistry The tool registry (Quay.io, GitLab, etc) of the tool whose access is being requested
    */
-  private getRequestEmailBody(path: string, toolRegistry: string): string {
-    return encodeURI(`I would like to request access to your Docker image ${path}. ` + `My user name on ${toolRegistry} is <username>`);
+  private static getRequestEmailBody(path: string, toolRegistry: string): string {
+    const imgProvider: string = toolRegistry ? toolRegistry : 'undefined';
+    return encodeURI(
+      `I would like to request access to your Docker image ${path}. ` + `My username on ${imgProvider} is <input username here>`
```

Comment:
You don't need to change this, but you could have gotten rid of the plus/concatenation operator, and just make it one string between backticks.